From b7c95475377660ebb6f0c96c6cb892d41238ada5 Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Mon, 6 Oct 2025 17:03:32 +0200
Subject: [PATCH] patch

---
 tests/conftest.py | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/tests/conftest.py b/tests/conftest.py
index bb44383..dee92ca 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -9,23 +9,23 @@ from langgraph.checkpoint.base import BaseCheckpointSaver
 from langgraph.store.base import BaseStore
 from tests.conftest_checkpointer import (
     _checkpointer_memory,
-    _checkpointer_postgres,
-    _checkpointer_postgres_aio,
-    _checkpointer_postgres_aio_pipe,
-    _checkpointer_postgres_aio_pool,
-    _checkpointer_postgres_pipe,
-    _checkpointer_postgres_pool,
-    _checkpointer_sqlite,
-    _checkpointer_sqlite_aio,
+    # _checkpointer_postgres,
+    # _checkpointer_postgres_aio,
+    # _checkpointer_postgres_aio_pipe,
+    # _checkpointer_postgres_aio_pool,
+    # _checkpointer_postgres_pipe,
+    # _checkpointer_postgres_pool,
+    # _checkpointer_sqlite,
+    # _checkpointer_sqlite_aio,
 )
 from tests.conftest_store import (
     _store_memory,
-    _store_postgres,
-    _store_postgres_aio,
-    _store_postgres_aio_pipe,
-    _store_postgres_aio_pool,
-    _store_postgres_pipe,
-    _store_postgres_pool,
+    # _store_postgres,
+    # _store_postgres_aio,
+    # _store_postgres_aio_pipe,
+    # _store_postgres_aio_pool,
+    # _store_postgres_pipe,
+    # _store_postgres_pool,
 )
 
 pytest.register_assert_rewrite("tests.memory_assert")
-- 
2.39.5 (Apple Git-154)

